### ISSUE TITLE
Stop the "type checking" notifications appearing

### ISSUE THREAD

**Issue by isaacabraham** [ionide/ionide-vscode-fsharp#2023]
In recent Ionide releases, typechecking notifications appear in the status bar *every single time* a file is type checked. This seems to be a continual thing - saving a single file causes all files to be type checked. To be honest, it's not the performance implications that I'm creating this issue for, but the fact that the constant animation in the status bar is very annoying. Even turning off Ionide notifications didn't help - I had to explicitly remove the Ionide status bar component, which means I've now lost all Ionide status updates (the rest of which are very useful).

**Comment by isaacabraham** [comment]
Sorry. It's the "Progress Message" notification that needed disabling in Code.

**Comment by TheAngryByrd** [comment]
I agree it can be annoying to have it pop up constantly. The flip side is seeing FSAC using your CPU and not knowing what it's crunching on.

I'll make this approved if someone wants to add a feature flag for this.
